ops/internal/manage/staging.go

```diff
@@ -215,3 +222,39 @@ func StagedSuperchainDefinition(rootP string) (*config.SuperchainDefinition, err
 
 	return sM, err
 }
+
+// isChainInDependencySet checks if a chain ID is in the supplied dependency set
+func isChainInDependencySet(chainID common.Hash, depSet *prestate.DependencySet) bool {
+	if depSet == nil {
+		return false
+	}
+	for id := range depSet.Dependencies {
+		if id == chainID.Hex() {
```

Comment:
Potential for a footgun here since the op-deployer struct uses `common.Hash` for chain Ids in some places and `string` in others. So if we compare hex encoding with decimal encoding we could miss a chain id match.